src/tools/perf/perftest_daemon.c

```diff
@@ -131,14 +424,14 @@ static ucs_status_t ucp_perf_daemon_init(ucp_perf_daemon_context_t *ctx)
     if (status != UCS_OK) {
         ucs_error("daemon failed to read UCP context: %s",
                   ucs_status_string(status));
-        goto err;
+        return status;
     }
 
     status = ucp_init(&ucp_params, config, &ctx->context);
     ucp_config_release(config);
     if (status != UCS_OK) {
         ucs_error("daemon failed to init UCP: %s", ucs_status_string(status));
-        goto err;
+        return status;
```

Comment:
why changed?

Reply:
To remove one extra goto label, that was used to just return the error.
IMO this way code is simpler, cause we don't need to scroll 50 lines below to see that function just returns error.

Reply:
our convention is that if a function has err handling flow with goto, all error flows direct to same place